app/ui-utils/client/lib/messageContext.js

```diff
@@ -30,6 +30,7 @@ export function messageContext({ rid } = Template.instance()) {
 			showreply: true,
 			showReplyButton: true,
 			hasPermissionDeleteMessage: hasPermission('delete-message', rid),
+			hasPermissionDeleteOwnMessage: hasPermission('delete-own-message'),
```

Comment:
is this being used somewhere?

Reply:
not yet, but `hasPermissionDeleteMessage` is not being used too, I just passing the context, and maybe after I could improve some stuffs.